modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\restful_example\Plugin\resource\Articles__1_0.
+ */
+
+namespace Drupal\restful_example\Plugin\resource;
+
+use Drupal\restful\Plugin\resource\ResourceEntity;
+use Drupal\restful\Plugin\resource\ResourceInterface;
+
+/**
+ * Class Articles
+ * @package Drupal\restful\Plugin\formatter
+ *
+ * @Resource(
+ *   name = "articles:1.0",
+ *   resource = "articles",
+ *   label = "Articles",
+ *   description = "Export the article content type with cookie authentication.",
+ *   authenticationTypes = TRUE,
+ *   authenticationOptional = TRUE,
+ *   dataProvider = {
+ *     "entityType": "node",
+ *     "bundles": {
+ *       "article"
+ *     },
+ *   },
+ *   majorVersion = 1,
+ *   minorVersion = 0
```

Comment:
I see where you are going. This is redundant if we are using `name = "articles:1.0",`.

I'd like to remove one or the other in the future. Any preferences?